usaspending_api/common/elasticsearch/elasticsearch_sql_helpers.py

```diff
@@ -26,3 +26,19 @@ def ensure_transaction_etl_view_exists(force: bool = False) -> None:
     view_sql = view_file_path.read_text()
     with connection.cursor() as cursor:
         cursor.execute(view_sql)
+
+
+def ensure_award_etl_view_exists(force: bool = False) -> None:
```

Comment:
The innards of this function and `ensure_transaction_etl_view_exists()` should be combined.

Example:

```python
def ensure_transaction_etl_view_exists(force: bool = False) -> None:
    view_name = settings.ES_TRANSACTIONS_ETL_VIEW_NAME
    view_file_path = settings.APP_DIR / "database_scripts" / "etl" / "transaction_delta_view.sql"
    ensure_view_exists(view_name, view_file_path, force)

def ensure_award_etl_view_exists(force: bool = False) -> None:
    view_name = settings.ES_AWARDS_ETL_VIEW_NAME
    view_file_path = settings.APP_DIR / "database_scripts" / "etl" / "award_delta_view.sql"
    ensure_view_exists(view_name, view_file_path, force)

def ensure_view_exists(view_name, view_sql, force):
    if verify_database_view_exists(view_name) and not force:
        return

    view_sql = view_file_path.read_text()
    with connection.cursor() as cursor:
        cursor.execute(view_sql)
```

Reply:
yeah that's fair